### PR TITLE
Fixed issues with some required modules

### PR DESCRIPTION
**Pull request by JGAntunes**
While trying to install and run ipfs I ran into some troubles, this should fix some of it.

### CODE REVIEW

**Comment by JGAntunes** [comment]
It also seems to me that the current build is failing due to this - https://github.com/ipfs/js-ipfs-unixfs/pull/8

**Comment by xicombd** [comment]
LGTM

File: src/cli/commands/config/replace.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const debug = require('debug')
 const path = require('path')
 const log = debug('cli:config')
 log.error = debug('cli:config:error')
-const utils = require('utils')
+const utils = require('../../utils')
```
Comment (diasdavid) [inline src/cli/commands/config/replace.js]: how was this not failing before? Could you add a test too @JGAntunes ? Thank you!

//cc @xicombd

File: package.json [hunk 2]
```diff
@@ -79,6 +78,7 @@
     "boom": "^3.1.1",
     "bs58": "^3.0.0",
     "debug": "^2.2.0",
+    "fs-blob-store": "^5.2.1",
```
Comment (diasdavid) [inline package.json]: good catch!

**Comment by diasdavid** [comment]
build is green now (for this branch as well), let me know if it is working for you.

File: src/cli/commands/config/replace.js (hunk 1 above)
Comment (xicombd) [inline src/cli/commands/config/replace.js]: No idea, I'll try to review the config commands later today, I think they're not working with the daemon running either :worried:

Comment (JGAntunes) [inline src/cli/commands/config/replace.js]: @diasdavid sure thing, will look at it later today also, so maybe I can help you out @xicombd.

**Comment by JGAntunes** [comment]
Seems good to me, I can now install and run the cli.

**Pull request merged**